extension.ts

```diff
@@ -227,6 +232,54 @@ async function createWebApp(context?: AzureImageTagNode | DockerHubImageTagNode)
   } else if (result.status === 'Cancelled') {
     throw new UserCancelledError();
   }
+  const website = wizard.createdWebSite;
+  let appUri = `https://${website.name}.scm.azurewebsites.net/docker/hook`;
+
+  if (context instanceof AzureImageTagNode) {
+    await createWebhookForWebApp(context, wizard, appUri);
+
+  } else {
+    // point to dockerhub to create a webhook
+    // http://cloud.docker.com/repository/docker/<registryName>/<repoName>/webHooks
+    const dockerhubPrompt: string = "Copy webapp endpoint and head to dockerhub";
```

Comment:
"browse" instead of "head" in both strings

Reply:
Changed. See #1085